services/gateway/shared/ready.js

```diff
@@ -16,56 +16,50 @@
 const BluebirdPromise = require('bluebird');
 const { MoleculerError } = require('moleculer').Errors;
 
+const logger = require('lisk-service-framework').Logger();
+const { getAppContext } = require('./appContext');
 const config = require('../config');
 
-const currentStatus = {
-	indexReadyStatus: false,
-	transactionStatsStatus: false,
-	feesStatus: false,
-	delegatesStatus: false,
+const currentSvcStatus = {
+	indexer: false,
+	connector: false,
+	fees: false,
+	market: false,
+	newsfeed: false,
+	'app-registry': false,
+	statistics: false,
 };
```

Comment:
would defining known microservices as a constant and using that to build this map be better?

Reply:
Thats was the initial idea but as we are defining `currentSvcStatus` with all known services, we can directly use it instead of defining another constant.